stack.yaml

```diff
@@ -78,7 +77,7 @@ extra-lib-dirs:
 #
 # Allow a newer minor version of GHC than the snapshot specifies
 # compiler-check: newer-minor
-# allow-newer: true
+allow-newer: true
```

Comment:
Why? This should not be enabled unless absolutely necessary. We want the dependency versions from the snapshot.

Reply:
I have enabled it after a build error, checking again...

Reply:
done